inc/class-s3-uploads.php

```diff
@@ -523,4 +523,43 @@ public function set_attachment_private_on_generate_attachment_metadata( array $m
 
 		return $metadata;
 	}
+
+	/**
+	 * Update the metadata headers on an attachment's metadata by copying it over itself.
+	 *
+	 * @param integer $attachment_id
+	 * @param string $values associative array of metadata. Existing values for a key will be overwritten.
+	 * @return void
+	 */
+	public function set_attachment_meta( int $attachment_id, array $values ): ?WP_Error {
+		$files = static::get_attachment_files( $attachment_id );
+		$locations = array_map( [ $this, 'get_s3_location_for_path' ], $files );
+		$s3 = $this->s3();
+		$commands = [];
```

Comment:
You used `array(...)` earlier in this PR. Best to stick to whatever s3 uploads is doing as a standard.

Reply:
OK